### PR TITLE
Added workshops controller + views

### PR DESCRIPTION
**Pull request by leefreemanxyz**
I was unsure as to what the planning was with regards to image uploads for workshops, so am open to suggestions here. Styling is currently non-existent.

### CODE REVIEW

File: app/controllers/workshops_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+class WorkshopsController < ApplicationController
+  def index
+    @workshops = Workshop.all
+  end
+
+  def show
+    @workshop = Workshop.find(params[:id])
```
Comment (Woulei) [inline app/controllers/workshops_controller.rb]: Maybe you want to reuse this code. Place it in a private method and use an before action?

File: app/controllers/workshops_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  def new
+    @workshop = Workshop.new
+  end
+
+  def edit
+    @workshop = Workshop.find(params[:id])
```
Comment (Woulei) [inline app/controllers/workshops_controller.rb]: Same here

File: app/controllers/workshops_controller.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    if @workshop.save
+      redirect_to workshop_path(@workshop), notice:"Workshop successfully created"
+    else
+      redirect_to new_workshop_path, alert:@workshop.errors.full_messages
+    end
+  end
+
+  def update
+    @workshop = Workshop.find(params[:id])
```
Comment (Woulei) [inline app/controllers/workshops_controller.rb]: Same

File: app/controllers/workshops_controller.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    if @workshop.update(workshop_params)
+      redirect_to workshop_path(@workshop), notice:"Workshop successfully updated"
+    else
+      render :edit
```
Comment (Woulei) [inline app/controllers/workshops_controller.rb]: No messages here?

File: app/helpers/workshops_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module WorkshopsHelper
+end
```
Comment (Woulei) [inline app/helpers/workshops_helper.rb]: Don't use it? => Delete it!   Sorry Anika started this.

File: spec/factories/workshops.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :workshop do
+    title "Feed reader"
+    description "blah blah blah"
```
Comment (Woulei) [inline spec/factories/workshops.rb]: Loving this

File: spec/helpers/workshops_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+# describe WorkshopsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe WorkshopsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (Woulei) [inline spec/helpers/workshops_helper_spec.rb]: Don't use it? => Delete it

File: app/controllers/workshops_controller.rb (hunk 1 above)
Comment (leefreemanxyz) [inline app/controllers/workshops_controller.rb]: Okay done.

File: app/controllers/workshops_controller.rb (hunk 2 above)
Comment (leefreemanxyz) [inline app/controllers/workshops_controller.rb]: Done.

File: app/controllers/workshops_controller.rb (hunk 3 above)
Comment (leefreemanxyz) [inline app/controllers/workshops_controller.rb]: Done.

File: app/helpers/workshops_helper.rb (hunk 5 above)
Comment (leefreemanxyz) [inline app/helpers/workshops_helper.rb]: Deleted.

File: spec/factories/workshops.rb (hunk 6 above)
Comment (leefreemanxyz) [inline spec/factories/workshops.rb]: Blah.

File: spec/helpers/workshops_helper_spec.rb (hunk 7 above)
Comment (leefreemanxyz) [inline spec/helpers/workshops_helper_spec.rb]: Deleted.

**Pull request merged**